prime-router/src/main/resources/metadata/hl7_mapping/receivers/STLTs/WA/WA-receiver-transform.yml

```diff
@@ -0,0 +1,154 @@
+    condition: 'Bundle.entry.resource.ofType(Patient).extension("http://ibm.com/fhir/cdm/StructureDefinition/local-race-cd").value[x].coding[0].code = ""'
+    value: [ '"NULLFL"' ]
+    hl7Spec: [ '%{PID}-10-3' ]
+
+  - name: wa-race-name-of-alternate-coding-system
+    condition: 'Bundle.entry.resource.ofType(Patient).extension("http://ibm.com/fhir/cdm/StructureDefinition/local-race-cd").value[x].coding[0].code = ""'
+    value: [ '"20090501"' ]
+    hl7Spec: [ '%{PID}-10-7' ]
+
+  - name: wa-primary-language-identifier
+    condition: 'Bundle.entry.resource.ofType(Patient).communication.exists().not() or Bundle.entry.resource.ofType(Patient).communication.language.exists().not() or Bundle.entry.resource.ofType(Patient).communication.language.code.exists().not()'
+    value: [ '"333"' ]
+    hl7Spec: [ '%{PID}-15-1' ]
+
+  - name: wa-primary-language-text
+    condition: 'Bundle.entry.resource.ofType(Patient).communication.exists().not() or Bundle.entry.resource.ofType(Patient).communication.language.exists().not() or Bundle.entry.resource.ofType(Patient).communication.language.code.exists().not()'
+    value: [ '"Unknown"' ]
+    hl7Spec: [ '%{PID}-15-2' ]
+
+  - name: wa-primary-language-name-of-coding-system
+    condition: 'Bundle.entry.resource.ofType(Patient).communication.exists().not() or Bundle.entry.resource.ofType(Patient).communication.language.exists().not() or Bundle.entry.resource.ofType(Patient).communication.language.code.exists().not()'
+    value: [ '"99WA_LANG"' ]
+    hl7Spec: [ '%{PID}-15-3' ]
+
+  - name: wa-primary-language-name-of-alternate-coding-system
+    condition: 'Bundle.entry.resource.ofType(Patient).communication.exists().not() or Bundle.entry.resource.ofType(Patient).communication.language.exists().not() or Bundle.entry.resource.ofType(Patient).communication.language.code.exists().not()'
+    value: [ '"1"' ]
+    hl7Spec: [ '%{PID}-15-7' ]
+
+  - name: wa-ethnic-group-identifier
+    condition: 'Bundle.entry.resource.ofType(Patient).extension("https://reportstream.cdc.gov/fhir/StructureDefinition/ethnic-group").exists().not()'
+    value: [ '"U"' ]
+    hl7Spec: [ '%{PID}-22-1' ]
+
+  - name: wa-ethnic-group-text
+    condition: 'Bundle.entry.resource.ofType(Patient).extension("https://reportstream.cdc.gov/fhir/StructureDefinition/ethnic-group").exists().not()'
+    value: [ '"Unknown"' ]
+    hl7Spec: [ '%{PID}-22-2' ]
+
+  - name: wa-ethnic-group-name-of-coding-system
+    condition: 'Bundle.entry.resource.ofType(Patient).extension("https://reportstream.cdc.gov/fhir/StructureDefinition/ethnic-group").exists().not()'
+    value: [ '"HL70189"' ]
+    hl7Spec: [ '%{PID}-22-3' ]
+
+  - name: wa-ethnic-group-name-of-alternate-coding-system
+    condition: 'Bundle.entry.resource.ofType(Patient).extension("https://reportstream.cdc.gov/fhir/StructureDefinition/ethnic-group").exists().not()'
+    value: [ '"HL7v2.5.1"' ]
+    hl7Spec: [ '%{PID}-22-7' ]
+
+  - name: wa-suppress-qst-for-aoe
```

Comment:
same comment as OH

Reply:
actually here I think we could ignore the condition and just blank OBX-29, although you'd have to check if that works if there are multiple OBXs

Reply:
I think it makes more sense with the condition.